openassessment/templates/openassessmentblock/oa_uploaded_file.html

```diff
@@ -12,15 +12,15 @@ <h5 class="{{ class_prefix }}__display__title">
 
     <div class="{{ class_prefix }}__display__file {% if not file_urls %}is--hidden{% endif %} submission__{{ file_upload_type }}__upload" data-upload-type="{{ file_upload_type }}">
         <div class="submission__answer__files">
-        {% for file_url, file_description, file_name in file_urls %}
+        {% for file_url, file_description, file_name, file_index in file_urls %}
             <div class="submission__answer__file__block submission__answer__file__block__{{ forloop.counter0 }}" {% if not file_url %} deleted {% endif %}>
```

Comment:
this line here as well as 28 also use the forloop counter. i vote we completely replace it.